rawx-apache2/src/rawx_repository.c

```diff
@@ -189,18 +189,18 @@ dav_rawx_get_resource(request_rec *r, const char *root_dir, const char *label,
 		resource->info->file_extension,
 		NULL);
 
-	/* init compression context structure if we are in get method */
-	if (r->method_number == M_GET && !ctx.update_only) {
-		resource_init_decompression(resource, conf);
-	}
-
 	/* Check the chunk's existence */
-	int flags = (r->method_number == M_GET ||
-			r->method_number == M_OPTIONS ||
-			r->method_number == M_DELETE)?
-				 RESOURCE_STAT_CHUNK_READ_ATTRS : 0;
-	if (r->method_number == M_PUT || r->method_number == M_POST)
+	int flags = 0;
+
+	if (r->method_number == M_GET) {
+		const char *value = apr_table_get(r->headers_in, "X-oio-xattr");
+		if (oio_str_parse_bool(value, TRUE))
+			flags |= RESOURCE_STAT_CHUNK_READ_ATTRS;
+	} else if (r->method_number == M_OPTIONS) {
+		flags |= RESOURCE_STAT_CHUNK_READ_ATTRS;
```

Comment:
Extended attributes are also used to complete the event during DELETE operations. Some of them are useless, but at least `container_id` and `content_id` (along with `chunk_id` and `volume_id`) are used to deindex the chunk from rdir.

Reply:
:man_facepalming: 

OK, got it.